7-extract_plier_pathways.R

```diff
@@ -362,52 +362,9 @@ if (length(jaccard_list) > 0) {
       "seed_index" = "L1"
     )
   
-  readr::write_tsv(
-    x = jaccard_df,
-    path = file.path(
-      res.dir,
-      str_c(file_identifier, "_PLIER_jaccard.tsv")
-    )
-  )
-  
-  # Plot results
-  
-  plot_filename = file.path(
-    plot.dir,
-    str_c(file_identifier, "_PLIER_jaccard.pdf")
-  )
-  
-  plot_data_filename = file.path(
-    plot.data.dir,
-    str_c(file_identifier, "_PLIER_jaccard.tsv")
-  )
-  
   readr::write_tsv(
     x = jaccard_df,
     path = plot_data.filename
   )
```

Comment:
Looking at the branch (i.e., not at the diff), I don't think this would run? Where is `plot_data.filename` coming from? Then what gets plotted in the plotting script?